docs/gaia/join-testnet.md

```diff
@@ -19,11 +19,11 @@ These instructions are for setting up a brand new full node from scratch.
 First, initialize the node and create the necessary config files:
 
 ```bash
-gaiad init --moniker <your_custom_moniker>
+gaiad init <your_custom_moniker>
```

Comment:
Should we consider using the node's hostname as the default moniker?

Reply:
I agree. Will amend accordingly

Reply:
Also make this arg optional?